test/app.js

```diff
@@ -3,6 +3,8 @@ var path = require('path');
 var assert = require('yeoman-assert');
 var helpers = require('yeoman-test');
 
+const titleize = word => `${word.charAt(0).toUpperCase()}${word.slice(1)}`;
```

Comment:
What does word stand for? I see its used in the mapping below but a comment might be helpful for quick reference.